UI/Selenium/UI.csproj

```diff
@@ -67,4 +65,19 @@
 		<ProjectReference Include="..\..\TestFramework\TestFramework.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <Folder Include="TestData" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <None Remove="TestData\NonAvailabilityHours.csv" />
```

Comment:
I don't think you need to remove and an embedded resource tag here. I would recommend removing the ItemGroup here and then ensure the css files are copied to the output dir